.github/workflows/continuous-integration.yml

```diff
@@ -81,18 +66,33 @@ jobs:
       - name: Install Python packages, excluding the unnecessary ones
         run: |
           python -m pip install --upgrade pip
-          sed -i '/\(pandas\|tqdm\|lint\|autopep8\|gunicorn\|django\-heroku\|uwsgi\)/d' requirements.txt
+          sed -i '/\(pandas\|tqdm\|gunicorn\|django\-heroku\|uwsgi\)/d' requirements.txt
+          echo coverage >> requirements.txt
```

Comment:
Would it be good to add `coverage` into `requirements.txt` directly, so coverage can be run as a part of `precommit`? (perhaps outside the scope of this PR)

Reply:
That's actually a great idea! I'll work on that as soon as this PR is merged.

Reply:
Created an issue for this: #361.